aws/biganimal-csp-preflight

```diff
@@ -481,13 +516,16 @@ function quota_suggest()
 FMT="%-21s %-37s %-8s %-8s %-11s %-8s %-11b\n"
 printf "$FMT" "Resource" "Quota Name" "Limit" "Used" "Required" "Gap" "Suggestion"
 printf "$FMT" "--------" "----------" "----" "-----" "--------" "---" "----------"
+[ "$shared_mgmt_eha_proxy_vcpus" -gt 0 ] && need_mgmt_vcpus=$((need_mgmt_vcpus + need_pg_eha_proxy_vcpus))
```

Comment:
In my understanding, the `need_mgmt_vcpus` means the default node pool CPUs. Why it needs to add the eha proxy vcpus?

Reply:
See https://github.com/EnterpriseDB/cloud-utilities/pull/68#discussion_r892495727

Reply:
Delete the duplicated comment.